lib/ulParser.js

```diff
@@ -239,13 +273,27 @@ function result(payload) {
  */
 function createCommandPayload(device, command, attributes) {
     function addAttributes(current, key) {
-        return current + '|' + key + '=' + attributes[key];
+        //let cmd = device && device.commands.find((att) => att.name === key);
+        let value = attributes[key];
+        // if (cmd && cmd.expression) {
+        //     let parser = iotAgentLib.dataPlugins.expressionTransformation;
+        //     let ctxt = device.staticAttributes.concat({key: value});
+        //     value = parser.applyExpression(cmd.expression, ctxt, device);
+        // }
```

Comment:
Leftover comments?

Reply:
fixed in https://github.com/telefonicaid/iotagent-ul/pull/550/commits/9908cba261d7f9b253df8a65010adba27ac273e2